### PR TITLE
chatbot name changes

### PR DESCRIPTION
**Pull request by souvikmaiti00**
Screenshot 2024-11-11 at 4 11 17 PM

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
 ---
 
 <div align="center">
-  <img src="./querius.png" alt="Querius Logo" width="300" height="300">
+  <img src="./chatbot.png" alt="Querius Logo" width="300" height="300">
 </div>
  
 <h2 align="center">IQuerius</h2>
```
Comment (PallaviKN7) [inline README.md:7]: IChatbot

File: frontend/src/locals/en.json [hunk 2]
```diff
@@ -10,17 +10,17 @@
   "searchInputPlaceholder": "Search Chats ...",
   "defaultErrorMessage": "I'm still learning! Can you rephrase your question in a different way, or maybe I can help you with something else?",
   "typingIndicatorMessage": "Go ahead, I'm listening...",
-  "welcomeStatementStart": "Hi, I am ",
-  "welcomeStatementEnd": ". How can I help?",
```
Comment (PallaviKN7) [inline frontend/src/locals/en.json:14]: check and remove unused constants
"querius": "Querius",
  "queriusLogo": "Querius Logo",

File: frontend/src/locals/en.json [hunk 3]
```diff
@@ -31,7 +31,7 @@
   "provideAnswerPrompt": "Please provide an answer",
   "usernameLengthInvalidPrompt": "Username must be at least 5 characters long and should not contain spaces in between.",
   "lastNameEmptyPrompt": "Last name cannot be empty and should not contain spaces in between.",
-  "createYourQueriusAccount": "Create your Querius account",
+  "createYourIBMStorageInsightsChatbotAccount": "Create your Chatbot using IBM Storage Insights APIs account",
```
Comment (PallaviKN7) [inline frontend/src/locals/en.json:34]: name cane be shortened

File: README.md (hunk 1 above)
Comment (souvikmaiti00) [reply README.md]: changed to IChatbot

File: frontend/src/locals/en.json (hunk 2 above)
Comment (souvikmaiti00) [reply frontend/src/locals/en.json:14]: removed

File: frontend/src/locals/en.json (hunk 3 above)
Comment (souvikmaiti00) [reply frontend/src/locals/en.json]: done

File: frontend/src/components/chatWindow/newChatWindow/newChatWindow.module.scss [hunk 4]
```diff
@@ -21,6 +21,21 @@
...
+  font-weight: 600;
+}
+
+.usingApis{
+  margin-top: 0.3rem;
+  margin-left: 0.3rem;
+  font-size: 1.25rem !important;
+}
+
+.title{
```
[the remark this answers is not in the document]
Comment (souvikmaiti00) [reply frontend/src/components/chatWindow/newChatWindow/newChatWindow.module.scss]: formatted the file

File: README.md [hunk 5]
```diff
@@ -1,7 +1,7 @@
 ---
 
 <div align="center">
-  <img src="./querius.png" alt="Querius Logo" width="300" height="300">
+  <img src="./chatbot.png" alt="Querius Logo" width="300" height="300">
```
[the remark this answers is not in the document]
Comment (souvikmaiti00) [reply README.md]: updated

**Review by PallaviKN7 — APPROVED** [review]

**Review by shubh1410 — APPROVED** [review]
